src/components/shared/levelSelect.tsx

```diff
@@ -1,44 +1,58 @@
+
+import {useState, useEffect} from 'react';
 import '../../styles/levelSelect.scss';
-import {useState} from 'react';
 import Dropdown, { Option } from 'react-dropdown';
-import { BsArrowUpShort, BsArrowDownShort } from 'react-icons/bs';
-
+import { BsCaretLeftFill , BsCaretRightFill, BsFillCaretUpFill, BsFillCaretDownFill } from 'react-icons/bs';
+import { useHistory } from 'react-router-dom';
 
-function LevelSelect(): JSX.Element {
-
-  const pages = ['1', '2', '3', '4', '5', '6', '7', '8', '9'];
+interface LevelSelectProps {
+  pages: string[];
+  locationPathName: string;
```

Comment:
nit: try to make prop names so intuitive that you don't need to read the code in order to know what to pass in
```suggestion
  pageOptions: string[];
  currentPage: string;
```